src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs

```diff
@@ -243,11 +301,60 @@ public void Pack(string nuspecPath, Func<string, string> nuspecPropertyProvider,
             }
             catch (Exception e)
             {
-                string packageType = packSymbols ? "symbol" : "lib";
+                string packageType = "lib";
+                if (creatingSymbolsPackage)
+                {
+                    packageType = "symbol";
+                }
+                else if (packSymbols)
+                {
+                    packageType = "packed";
+                }
                 Log.LogError($"Error when creating nuget {packageType} package from {nuspecPath}. {e}");
             }
         }
 
+        private Manifest TransformManifestToPackedPackageManifest(Manifest manifest)
```

Comment:
We need to change the Id of the package, update the dependencies, and change the runtime graph file which gets included in the identity package to handle the "transport" line of packages.